src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -191,7 +191,7 @@ def configure_container_dict(
             "PYFLUENT_CONTAINER_MOUNT_SOURCE", pyfluent.CONTAINER_MOUNT_SOURCE
         )
     if not mount_source:
-        mount_source = pyfluent.USER_DATA_PATH if file_transfer_service else os.getcwd()
+        mount_source = pyfluent.EXAMPLES_PATH if file_transfer_service else os.getcwd()
```

Comment:
I would make a similar comment here regarding `pyfluent.EXAMPLES_PATH`.

Reply:
I think we can default `mount_source` to the current working directory in both places. `EXAMPLES_PATH` is needed for unitests or doc examples which we're setting via the module/env var.
@hpohekar Can you please confirm?

Reply:
@seanpearsonuk @mkundu1 

No. It's depend on whether we are using file transfer service or not, as explained in above comments.